src/loadmill.ts

```diff
@@ -21,7 +24,7 @@ program
     .option("-v, --verbose", "Print out extra information for debugging.")
     .option("-r, --report", "Print out Test Suite Flow Runs report when the suite has ended.")
     .option("--colors", "Print test results in color")
-    .option("-c, --local", "Execute functional test synchronously on local machine. This flag trumps load-test and async options")
+    .option("-c, --local", "Execute functional test synchronously on local machine. This flag trumps load-test option")
```

Comment:
Maybe its time to rename "trumps" to be more readable for 99.9% of the developers :)

Reply:
What would you suggest?

Reply:
prevents ?

Reply:
It's not the same meaning.